Cargo.toml

```
@@ -832,7 +832,7 @@ linked-hash-map = { version = "0.5.4" }
 linked_hash_set = { version = "0.1.4" }
 linregress = { version = "0.5.1" }
 lite-json = { version = "0.2.0", default-features = false }
-litep2p = { version = "0.6.2" }
+litep2p = { version = "0.7.0", default-features = false }
```

I don't think we need this, as all extra transports are disabled by default.
```suggestion
litep2p = { version = "0.7.0" }
```